tensorflow_encrypted/tensor/int100.py

```diff
@@ -59,7 +61,7 @@ class Int100Tensor(object):
     def __init__(
         self,
         native_value: Optional[Union[np.ndarray, tf.Tensor]],
-        decomposed_value: Optional[Union[List[np.ndarray], List[tf.Tensor]]] = None
+        decomposed_value: Optional[Union[List[np.ndarray], List[tf.Tensor], List[int]]] = None
```

Comment:
why the `List[int]`?

Reply:
something was complaining not sure what anymore! will remove for now

Reply:
i'm guessing strange things might happen if you pass a list of ints -- did you try?

Reply:
ints are getting passed in here: https://github.com/mortendahl/tf-encrypted/pull/128/files#diff-0697318ec50fe33b892f344d6d00b5a4L89 does the zeros function work?

Reply:
ah good catch, that's probably a bug, should be wrapped in np.array i think